apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Input } from "~/libs/components/components.js";
+import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
+import {
+	type ProjectCreateRequestDto,
+	projectCreateValidationSchema,
+} from "~/modules/projects/projects.js";
+
+import { DEFAULT_PROJECT_CREATE_PAYLOAD } from "./libs/constants/constants.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	onSubmit: (payload: ProjectCreateRequestDto) => void;
+};
+
+const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
+	const { control, errors, handleSubmit } = useAppForm<ProjectCreateRequestDto>(
+		{
+			defaultValues: DEFAULT_PROJECT_CREATE_PAYLOAD,
+			validationSchema: projectCreateValidationSchema,
+		},
+	);
+
+	const handleFormSubmit = useCallback(
+		(event_: React.BaseSyntheticEvent): void => {
+			void handleSubmit((formData: ProjectCreateRequestDto) => {
+				onSubmit(formData);
+			})(event_);
+		},
+		[handleSubmit, onSubmit],
+	);
+
+	const isButtonDisabled = Boolean(errors.name) || Boolean(errors.description);
```

Comment:
I meant that we don't need to disable button because of validation errors at all. User should be able to click on it, so it will trigger the validation again and appropriate messages will be displayed
```suggestion
```

Reply:
Oh, get it.
I didn't get that first time ;) The thing that confused me is that in Figma the button was disabled
![image](https://github.com/user-attachments/assets/3091b25d-09ae-4c9c-bd04-489c448cadc1)

Should I remove `isDisabled` prop of `Button` component? Not sure, if we will need this prop, if we have field errors instead.
P.s. I will remove it for now

Reply:
We will need disabled state, for example, for profile form, we have different logic. And there we will disable button until nothing changed. But it will be added here #107